Bottom: cac6d024e364dc5589ecd004f8311a18602c941f
Top:    117ed23ffa0a44d2fe5626429f9856046cb31c4f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-13 20:10:29 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 507aa78..5a44fb8 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
